docs/advanced/graphql.md

```diff
@@ -3,7 +3,7 @@ id: graphql
 title: GraphQL API (Beta)
 ---
 
-> **Note** 
+> **Note**  
```

Comment:
```suggestion
> **Note**
```

Might want a pre-commit there? I believe it works on npm. Otherwise we will end up with more of these and some editors will trail them automatically, creating _unnecessary_ diffs.

```yaml
-   repo: https://github.com/pre-commit/pre-commit-hooks
    rev: v2.2.3
    hooks:
    -   id: trailing-whitespace
    -   id: end-of-file-fixer
    -   id: flake8
```

Reply:
It may be a good idea. Unfortunately, I won't have time now to configure it. Correct me if I'm wrong, but I thought pre-commit only works when creating commits from shell? None of our writers use shell but the UI tools for git.

Reply:
It works but they might run into weird behaviors from the editors as they most often don't expect such responses.